sentry-types/Cargo.toml

```diff
@@ -30,3 +30,4 @@ thiserror = "1.0.15"
 time = { version = "0.3.5", features = ["formatting", "parsing"] }
 url = { version = "2.1.1", features = ["serde"] }
 uuid = { version = "1.0.0", features = ["v4", "serde"] }
+chrono = { version = "0.4.22",  features = ["serde"] }
```

Comment:
I wouldn’t really want to add back chrono.
We removed that in https://github.com/getsentry/sentry-rust/pull/409 as it was unmaintained for a long time. I think they found new maintainers in the meantime, but we should still get along fine with just `time`.

Reply:
fair enough, but I'd need a way to connvert `SystemTime` to `ISO 8601` string representation to comply to what the backend expects (see example [here](https://github.com/getsentry/relay/pull/1462/files#diff-aacb2c0a29425780822d9231b633ab596c0a141611568e91de64955a8df0b86cR16)) and I'm not sure there is a way to do that with `time`.

ps.  checking at their Github release page I can see they started releasing new versions and fixing stuff from July onward.

Reply:
RUSTSEC-2020-0071 matches again now. https://github.com/getsentry/sentry-rust/issues/371 should be re-opened.